Versionsmigration/5.4-6.0/xplangml-migration-54-60b.halex.alignment.xml

```diff
@@ -12432,7 +12446,7 @@ if (refLegende){&#13;
         </complexParameter>
         <complexParameter name="script">
             <core:text xmlns:core="http://www.esdi-humboldt.eu/hale/core" xml:space="preserve">
-if(_source.links.BP_Bereich.BP_Plan.first()){&#13;
+if(_source.links.BP_Bereich.BP_Plan.first() &amp;&amp; _source.p.refText.first()){&#13;
```

Comment:
With this additional condition here "XP_TextAbschnitt" is only created if "refText" contains a value. That's correct.
But we would need to add this condition also for the Groovy Joins of FP/LP/RP/SO_Plan to XP_TextAbschnitt